app/frontend/src/components/admin/inspection/LocationEdit.vue

```diff
@@ -0,0 +1,160 @@
+import { mapGetters, mapMutations, mapActions } from 'vuex';
+
+import Location from '@/components/form/Location.vue';
+import ipcService from '../../../services/ipcService';
+
+export default {
+  name: 'LocationEdit',
+  components: {
+    Location,
+  },
+  props: {
+    ipcPlanId: {
+      required: true,
+      type: String
+    },
+  },
+  data() {
+    return {
+      dialog: false,
+      updateValid: false,
+      reviewMode: false,
+      updateComplete: false,
+      updateError: '',
+      startDateMenu: false,
+      endDateMenu: false
+    };
+  },
+  computed: {
+    // use local 'edit' store
+    ...mapGetters('edit', ['business', 'location']),
+    ...mapMutations('edit', ['updateLocation']),
+
+    // Business data
+    businessName: {
+      get() { return this.business.name; },
+    },
+
+    // Location
+    // test - use local store in this component - doesnt work as i hoped
+    locationCity: {
+      get() {
+        console.log('in edit component, get locationCity: ', this.location.city);
+        return this.location.city;
+      },
+      set(value) {
+        console.log('in set edit component, get locationCity: ', this.location.city);
+        this.updateLocation({['city']: value});
+      }
+    },
+  },
+  methods: {
+    ...mapActions('form', ['getForm']),
+
+    openDialog() {
+      document.querySelectorAll('.review-form input, .review-form .v-select').forEach(q => {
+        q.setAttribute('readonly', 'false');
+        q.style.pointerEvents = 'none';
+      });
+    },
+
+    async submit() {
+      if(this.$refs.form.validate()) {
+        await this.updateForm();
+        if (this.updateComplete) {
+          this.dialog = false;
+          // reload form data to show in parent component;
+          this.getForm(this.ipcPlanId);
+        }
+      } else {
+        const modal = document.querySelector('.v-dialog--active');
+        modal.scrollBy(0, -500);
+      }
+    },
+
+    async updateForm() {
+      try {
+        const body = {
+          business: this.business,
+          contacts: this.contacts,
+          ipcPlan: this.ipcPlan,
+          covidContact: this.covidContact,
+          location: this.location
+        };
+        const response = await ipcService.updateIPCContent(body);
+        if (!response.data) {
+          throw new Error('No response data from API while submitting update');
+        }
+        this.updateComplete = true;
+      } catch (error) {
+        console.error(`Error submitting update: ${error}`); // eslint-disable-line no-console
+        this.updateError = 'An error occurred while attempting to update the form. Please try again.';
+      }
+    },
+
+    cancelUpdate(){
+      this.dialog = false;
+      // reload form data in parent component - from the global 'from' state module
+      this.getForm(this.ipcPlanId);
+    },
+
+  },
+  // lazy-load a local copy of the 'form' store into the 'edit' namespace
+  beforeCreate() {
```

Comment:
needed to use beforeCreate() instead.
So i can now use the form module in a 'local' namespace but i havent found a way to override the getters and setters in the child Location component.